parser/parser_test.go

```diff
@@ -1223,3 +1223,17 @@ func TestParse(t *testing.T) {
 		})
 	}
 }
+
+func TestExpressionSizeCodePointLimit(t *testing.T) {
```

Comment:
Can you also add a test for a negative valued limit?

Reply:
Done.